packages/hardhat-plugin/test/use-ignition-project.ts

```diff
@@ -14,6 +10,15 @@ import { IgnitionHelper } from "../src/ignition-helper";
 import { clearPendingTransactionsFromMemoryPool } from "./execution/helpers";
 import { waitForPendingTxs } from "./helpers";
 
+declare module "mocha" {
```

Comment:
@kanej I tried to type these helpers to be able to debug some things, but I don't think the other `deploy` helper can be typed.

To be honest, I find creating this kind of helpers really hard to reason, especially since they have type `any` unless you type them.

Reply:
I have pushed a temporary typing. I split out the two deploys (file based and ephemeral), one is now `runControlledDeploy` and is mainly used in the execution that is currently skipped.

I have no problem pulling these helpers out/replacing the tests with a different structure. We talked about moving the execution tests currently skipped into core, which may simplify the integration tests enough this issue goes away.